kit/ChildSession.cpp

```diff
@@ -2481,13 +2481,13 @@ bool ChildSession::renderNextSlideLayer(SlideCompressor &scomp,
             std::string json = jsonMsg;
             Poco::JSON::Parser parser;
             Poco::JSON::Object::Ptr root;
+            root = parser.parse(json).extract<Poco::JSON::Object::Ptr>();
+            root->set("cacheKey", cacheKey);
+            json = JsonUtil::jsonToString(root);
```

Comment:
We parse the json, set a thing, turn the parsed json back to a string and if enableexperimental parse it again and do a similar thing. 

Could we just move the json=jsonToString to the line after the close of the EnableExperimental block and not reparse inside EnableExperimental, but instead just work on the already parsed json?

And I suppose might as well make it: a single line of:

Poco::JSON::Object::Ptr root = parser.parse(json).extract<Poco::JSON::Object::Ptr>();